README.md

```diff
@@ -1,6 +1,25 @@
-# @ipld/dag-pb
+# @ipld/dag-pb <!-- omit in toc -->
 
-An implementation of the [DAG-PB spec](https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-pb.md) for JavaScript designed for use with [multiformats](https://github.com/multiformats/js-multiformats) or via the higher-level `Block` abstraction in [@ipld/block](https://github.com/ipld/js-block).
```

Comment:
why did this get removed? does the tooling just expect the intro section to be badges?

Reply:
No particular reason - the readme is parsed and standardised by aegir to have the same badges, toc, license, install instructions etc everywhere.

It's a work in progress though, more than happy to merge fixes to [the implementation](https://github.com/ipfs/aegir/tree/master/src/check-project/readme).